extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -124,6 +129,46 @@ html, body {
   max-width: 1084px;
 }
 
+.conversation-skin-help-card-container.ng-enter,
+.conversation-skin-help-card-container.ng-leave {
+  opacity: 0;
+  transform: translateX(200px);
+}
+.conversation-skin-help-card-container {
+  opacity: 1;
+  position: fixed;
+  right: 30px;
+  top: 80px;
+  transform: translateX(0);
+  transition: all 500ms;
```

Comment:
500ms feels a bit slow, I would change this to 350ms.

Actually, there's an issue right now– if you click on the map, there is a delay of about a second before anything happens. We added this to make it feel like Oppia is 'thinking' when you type an answer in; however, on supplemental interactions, there are no 'thinking dots' at the moment, so it just seems frozen.

Would it be possible to skip the forced delay for supplemental interactions? The 350ms fade-in animation might be enough of a delay already in those cases.